Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WinForms/Microsoft.Toolkit.Win32.UI.Controls.WinForms.csproj

```diff
@@ -46,40 +37,28 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Update="DesignerUI.Designer.cs">
-      <DependentUpon>DesignerUI.resx</DependentUpon>
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-    </Compile>
-    <Compile Update="WinForms\WebView\WebView.BaseProperties.cs">
+    <ProjectReference Include="..\Microsoft.Toolkit.Win32.UI.Interop.WinForms\Microsoft.Toolkit.Win32.UI.Interop.WinForms.csproj" />
```

Comment:
Why in a separate project? Separate project = separate DLL = additional overhead to verify the DLL and bind at runtime